lib/external/sentry.js

```diff
@@ -128,6 +127,11 @@ const init = (config) => {
       return null; // we have a user-space problem.
     }
   });
+
+  const { DOMAIN } = process.env;
+  const username = DOMAIN?.endsWith('.getodk.cloud') ? DOMAIN : require('os').hostname();
```

Comment:
We're trying where possible to avoid having logic specific to ODK Cloud in the codebase. Could we use the hostname all the time? It's set to a useful value in ODK Cloud, I believe.

Reply:
:+1:  updated